apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/v1/controller/ItemController.java

```diff
@@ -60,6 +60,10 @@ public OpenItemDTO createItem(@PathVariable String appId, @PathVariable String e
       throw new BadRequestException("User " + item.getDataChangeCreatedBy() + " doesn't exist!");
     }
 
+    if( (StringUtils.isEmpty(item.getComment()) ? 0 : item.getComment().length())  > 64){
+      throw new BadRequestException("The item comment length is greater than 64");
```

Comment:
how about `Comment length should not exceed 64 characters` so that it's the same as the English translation in en.json.

Reply:
ok